<ui version="4.0" >
 <class>modCalcSidTimeDlg</class>
 <widget class="QWidget" name="modCalcSidTimeDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Sidereal Time</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="interact" >
      <attribute name="title" >
       <string>Interactive Mode</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="2" column="0" colspan="3" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>391</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item rowspan="2" row="3" column="2" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1" colspan="2" >
          <widget class="KPushButton" name="LocationButton" >
           <property name="text" >
            <string>Greenwich, United Kingdom</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" >
          <widget class="QLabel" name="DateLabel" >
           <property name="text" >
            <string>Date:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1" >
          <widget class="QTimeEdit" name="ST" />
         </item>
         <item row="3" column="1" >
          <widget class="QTimeEdit" name="LT" />
         </item>
         <item row="4" column="0" >
          <widget class="QLabel" name="SiderealTimeLabel" >
           <property name="text" >
            <string>Sidereal time:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" >
          <widget class="QLabel" name="LocalTimeLabel" >
           <property name="text" >
            <string>Local time:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2" >
          <layout class="QHBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <widget class="ExtDateEdit" name="Date" />
           </item>
           <item>
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="LocationLabel" >
           <property name="text" >
            <string>Location:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="batch" >
      <attribute name="title" >
       <string>Batch Mode</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_4" >
         <property name="title" >
          <string>Select Parameters in Input File</string>
         </property>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <layout class="QGridLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item row="0" column="0" >
             <widget class="QCheckBox" name="UTCheckBatch" >
              <property name="toolTip" >
               <string>If checked, value will be read from input file. If not, value will be read from adjacent box</string>
              </property>
              <property name="text" >
               <string>UT:</string>
              </property>
              <property name="checked" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="1" >
             <widget class="QTimeEdit" name="OutputTimeBoxBatch" />
            </item>
            <item row="1" column="0" >
             <widget class="QCheckBox" name="STCheckBatch" >
              <property name="enabled" >
               <bool>true</bool>
              </property>
              <property name="toolTip" >
               <string>If checked, value will be read from input file. If not, value will be read from adjacent box</string>
              </property>
              <property name="text" >
               <string>LST:</string>
              </property>
              <property name="checked" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <widget class="QTimeEdit" name="InputTimeBoxBatch" />
            </item>
           </layout>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QGridLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item row="0" column="1" >
             <widget class="ExtDateEdit" name="DateBoxBatch" />
            </item>
            <item row="0" column="0" >
             <widget class="QCheckBox" name="DateCheckBatch" >
              <property name="enabled" >
               <bool>true</bool>
              </property>
              <property name="toolTip" >
               <string>If checked, value will be read from input file. If not, value will be read from adjacent box</string>
              </property>
              <property name="text" >
               <string>Date:</string>
              </property>
              <property name="checked" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="1" >
             <widget class="dmsBox" name="LongitudeBoxBatch" />
            </item>
            <item row="1" column="0" >
             <widget class="QCheckBox" name="LongCheckBatch" >
              <property name="toolTip" >
               <string>If checked, value will be read from input file. If not, value will be read from adjacent box</string>
              </property>
              <property name="text" >
               <string>Longitude:</string>
              </property>
              <property name="checked" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5" >
         <property name="title" >
          <string>Select Parameters for Output File</string>
         </property>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QRadioButton" name="AllRadioBatch" >
            <property name="text" >
             <string>All parameters</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="InputRadioBatch" >
            <property name="text" >
             <string>Only parameters in input file</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_6" >
         <property name="title" >
          <string>Select Filenames</string>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="1" column="1" >
           <widget class="KUrlRequester" name="OutputFileBoxBatch" />
          </item>
          <item row="0" column="1" >
           <widget class="KUrlRequester" name="InputFileBoxBatch" />
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_4" >
            <property name="text" >
             <string>Output file:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" >
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>Input file:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="KPushButton" name="RunButtonBatch" >
           <property name="text" >
            <string>Run</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>ExtDateEdit</class>
   <extends>QSpinBox</extends>
   <header>extdatetimeedit.h</header>
  </customwidget>
  <customwidget>
   <class>dmsBox</class>
   <extends>KLineEdit</extends>
   <header>widgets/dmsbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
